### PR TITLE
Update channel-settings.rst

### PR DESCRIPTION
**Pull request by wiersgallak**
Docs for https://github.com/mattermost/mattermost-server/pull/14214

### CODE REVIEW

File: source/help/settings/account-settings.rst [hunk 1]
```diff
@@ -13,17 +13,17 @@ Settings to configure name, username, nickname, email and profile picture.
...
+Full names appear in the Direct Messages member list and team management modal. By default, you will receive mention notifications when someone types your first name. Entering a full name is optional. Please note that `some unicode characters <https://www.w3.org/TR/unicode-xml/#Charlist>`_ are not supported. 
 
 Username
 ~~~~~~~~
 
-Usernames are unique identifiers appearing next to all posts. Pick something easy for teammates to recognize and recall. By default, you will receive mention notifications when someone types your username. In order to maintain message integrity, changing your username does not update @mentions in messages already posted. 
+Usernames are unique identifiers appearing next to all posts. Pick something easy for teammates to recognize and recall. By default, you will receive mention notifications when someone types your username. In order to maintain message integrity, changing your username does not update @mentions in messages already posted. Please note that `some unicode characters <https://www.w3.org/TR/unicode-xml/#Charlist>`_ are not supported.
```
Comment (esethna) [inline source/help/settings/account-settings.rst:21]: Thanks @wiersgallak, wondering if there a source we can link that is more readable from the perspective of an end user given the target audience for this section of the docs? ie I think U+0340 is a backtick?

Comment (esethna) [reply source/help/settings/account-settings.rst]: This was a screenshot from QA testing on the PR so I believe it's already implemented in the product. So we can probably just add this note here in the docs to match

Comment (wiersgallak) [reply source/help/settings/account-settings.rst]: @esethna Thanks for clarifying and great idea.  Please review the update I made in the username session.

Comment (esethna) [reply source/help/settings/account-settings.rst]: Looks good, thanks Katie!

**Pull request merged**